test/README.asciidoc

```diff
@@ -12,12 +12,14 @@ Source structure
 │   └── …
 └── compose
     └── …
+        ├── [rc]         → configuration
         ├── cmd          → command
         ├── [in]         → start file
         ├── [out]        → end file
         ├── [selections] → selection contents
         ├── [state]      → selection states
-        └── [rc]         → configuration
+        ├── [display]    → out display in JSON format
```

Comment:
That is out of date, and should be named `ui-out`, there is also a `ui-in` that can contain json commands as they would be sent by a json-ui.